.github/workflows/workflow.yml

```diff
@@ -104,6 +103,19 @@ jobs:
         run: cargo build --release --locked
       - name: Version info
         run: ./target/release/basilisk --version
+      - name: Copy Binary for docker image creation
+        run: cp target/release/basilisk docker/
+      - name: install docker
+        run: sudo snap install docker
+      - name: Sleep for 30 seconds
+        run: sleep 30s
+      - name: build docker image
+        run: sudo docker build -t basilisk:latest docker
+      - name: get ecr credentials
+        run: aws ecr get-login-password --region eu-west-1 | sudo docker login --username AWS --password-stdin 601305236792.dkr.ecr.eu-west-1.amazonaws.com
+      - name: push image to ecr repository
+        run: sudo docker tag basilisk:latest 601305236792.dkr.ecr.eu-west-1.amazonaws.com/basilisk
+      - run: sudo docker push 601305236792.dkr.ecr.eu-west-1.amazonaws.com/basilisk:latest  
```

Comment:
don't think you'd have to use SUDO on a CI runner, might be potentionally unsafe.

Reply:
Agree but it's a temporary machine so